packages/dotlottie-js/src/tests/dotlottie-js-browser.spec.ts

```diff
@@ -539,9 +542,28 @@ describe('fromURL', () => {
     expect(fetchSpy).toHaveBeenCalledWith(animationURL);
     expect(dotLottie.animations.length).toBe(1);
     expect(dotLottie.animations[0]?.id).toEqual(manifest.animations[0]?.id as string);
-    expect(dotLottie.animations[0]?.data).toEqual(animationData as AnimationData);
+    expect(dotLottie.animations[0]?.data).toEqual(animationData as unknown as Animation);
     expect(dotLottie.manifest).toEqual(manifest as Manifest);
   });
+
+  it('loads a dotLottie with non-default settings from a URL and verifies the animation settings', async () => {
+    const fetchSpy = spyOn(typeof window === 'undefined' ? global : window, 'fetch').and.returnValue(
+      Promise.resolve(new Response(editedDotlottieAnimation)),
+    );
+
+    const animationURL = 'https://lottiefiles.fake/animation/animation.lottie';
+
+    let dotlottie = new DotLottie();
+
+    dotlottie = (await dotlottie.fromURL('https://lottiefiles.fake/animation/animation.lottie')) as DotLottie;
```

Comment:
why do we need the `as DotLottie`, shouldn't fromUrl return a dotLottie instance ?

Reply:
have removed